sdk/storage/azure-storage-file-datalake/tests/test_file_system.py

```diff
@@ -916,6 +916,30 @@ def test_list_paths_recursively(self, **kwargs):
         # there are 24 subpaths in total
         assert len(paths) == 24
 
+    @DataLakePreparer()
+    @recorded_by_proxy
+    def test_list_paths_begin_from(self, **kwargs):
+        datalake_storage_account_name = kwargs.pop("datalake_storage_account_name")
+        datalake_storage_account_key = kwargs.pop("datalake_storage_account_key")
+
+        self._setUp(datalake_storage_account_name, datalake_storage_account_key)
+        # Arrange
+        file_system = self._create_file_system()
+        for i in range(0, 6):
+            file_system.create_directory("dir{}".format(i))
+
+            # create a subdirectory under the current directory
+            subdir = file_system.get_directory_client("dir{}".format(i)).create_sub_directory("subdir")
```

Comment:
nit: `create_directory` above returns a directory client you can use instead of making a new one.